lib/browser/guest-window-manager.js

```diff
@@ -47,8 +47,19 @@ const mergeBrowserWindowOptions = function (embedder, options) {
     options.webPreferences = {}
   }
   if (embedder.browserWindowOptions != null) {
+    let parentOptions = embedder.browserWindowOptions
+
+    // if parent's visibility is available, that overrides 'show' flag (#12125)
+    const win = BrowserWindow.fromWebContents(embedder.webContents)
+    if (win != null) {
+      const show = win.isVisible()
+      if (typeof show === 'boolean') {
```

Comment:
The docs say that `isVisible()` always returns a boolean
https://github.com/electron/electron/blob/master/docs/api/browser-window.md#winisvisible

Reply:
Yeah didn't really query this, but looking at it, it can only ever be true/false right?